docs/en/latest/installation-guide.md

```diff
@@ -216,6 +216,12 @@ brew services start etcd
 
 ### Configuring APISIX
 
+If you install APISIX via Docker, please exec into the apisix pod first:
```

Comment:
Docker? The command is `kubectl`.

Reply:
@tokers I meant this type of installation using docker: 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/61597896/208391871-c1cc7d47-fc41-4551-9bce-def0260e3c86.png">

Reply:
I think the appropriate prompt would be: "If you have done the kubernetes/docker/helm-based installation" WDYT?

Reply:
Then why do users need to execute `kubectl`? It should be `docker`.

Reply:
@tokers Please see this suggestion to this problem: https://github.com/apache/apisix/pull/8536#discussion_r1051960354

Reply:
Got it.